bootstrap

```diff
@@ -147,6 +147,8 @@ if [ -e ./$VENV ]  && [ ! -e ./$VENV/lib/python*/no-global-site-packages.txt ];
     rm -rf ./$VENV
 fi
 
+export LC_ALL=en_US.UTF-8
```

Comment:
Don't we want to export LANG here as well?

Reply:
good point. will export it if any of us needs it. i think `bootstrap` is used by end user. so a localized output might be expected.

while `tox` is run by our CI in general, so i want to ensure that the output is in English.